frontend/src/components/CarCard.jsx

```diff
@@ -10,6 +10,7 @@ function CarCard(props) {
   const dispatch = useDispatch();
   const userVehicle = useSelector(state => state.user.carData);
 
+  // // useEffect et axios à retirer quand les routes sécurisées seront en place
```

Comment:
Don't leave instructions to future-you : either do it now, or add the task to your backlog